src/Standard/TIP-4/4.md

```diff
@@ -84,39 +84,49 @@ Every `Storage` contract must implement `TIP4_4Storage`
 pragma ton-solidity >= 0.58.0;
 
 interface TIP4_4Storage {
-    function fill(uint8 id, bytes chunk, address gasReceiver) external;
+    function fill(uint32 id, bytes chunk) external;
```

Comment:
`uint8` > `uint32` is good ✅

Removing `address gasReceiver` is bad ❌
`gasReceiver` is required for internal messages. Need to know where to send remaining gas after each transaction. Example:
* Balance of `Storage` contract is **0.1 EVER**
* Call `fill()` with **1 EVER** using internal message **first time**. **0.1 EVER** spend for fee fee. **0.9 EVER** back to `gasReceiver`
* Call `fill()` with **1 EVER** using internal message **last time**. **0.1 EVER** spend for fee. **0.9 EVER** back to `gasReceiver`
* Balance of `Storage` contract still is **0.1 EVER**

`gasReceiver` is also required for external messages. Need to know where to send remaining gas from Storage contract balance each last `fill()` call. Example:
* Balance of `Storage` contract is **1 EVER**
* Call `fill()` with using external message **first time**. **0.1 EVER** spend for fee fee. * Balance of `Storage` contract is **0.9 EVER**
* Call `fill()` with using external message **last time**. **0.1 EVER** spend for fee. **0.7 EVER** back to `gasReceiver`
* Balance of `Storage` contract still is **0.1 EVER**